ImageManipulator/ImageManipulator/ImageManipulator/AppDelegate.swift

```diff
@@ -0,0 +1,82 @@
+//
+//  AppDelegate.swift
+//  ImageManipulator
+//
+//  Created by Lambda_School_Loaner_268 on 5/4/20.
+//  Copyright © 2020 Lambda. All rights reserved.
+//
+
+import UIKit
+import CoreData
+
+@UIApplicationMain
+class AppDelegate: UIResponder, UIApplicationDelegate {
+
+
+
+    func application(_ application: UIApplication, didFinishLaunchingWithOptions launchOptions: [UIApplication.LaunchOptionsKey: Any]?) -> Bool {
+        // Override point for customization after application launch.
+        return true
+    }
+
+    // MARK: UISceneSession Lifecycle
+
+    func application(_ application: UIApplication, configurationForConnecting connectingSceneSession: UISceneSession, options: UIScene.ConnectionOptions) -> UISceneConfiguration {
+        // Called when a new scene session is being created.
+        // Use this method to select a configuration to create the new scene with.
+        return UISceneConfiguration(name: "Default Configuration", sessionRole: connectingSceneSession.role)
+    }
+
+    func application(_ application: UIApplication, didDiscardSceneSessions sceneSessions: Set<UISceneSession>) {
+        // Called when the user discards a scene session.
+        // If any sessions were discarded while the application was not running, this will be called shortly after application:didFinishLaunchingWithOptions.
+        // Use this method to release any resources that were specific to the discarded scenes, as they will not return.
+    }
+
+    // MARK: - Core Data stack
+
```

Comment:
I think this is the code that comes along if you have "Core Data" checked when you make a new project. I think I'd actually recommend *never* having this checked, as it kind of railroads you into using it a certain way to a certain extent. Instead I think it'd be better to add a custom Core Data Stack when you need it. You might consider an Xcode snippet for the Core Data Stack to make it extra convenient (though it involves a bit of setup up front.